frontend/src/components/Dashboard/CrossingListPage/DashboardCrossingListItem/StatusToggle/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
Having this file named index.js doesn't fit with existing naming patterns. Do you like this pattern better? 

It's cool that it allows us to 
```
 import StatusToggle from 'components/Dashboard/CrossingListPage/DashboardCrossingListItem/StatusToggle';
```
instead of 
```
 import StatusToggle from 'components/Dashboard/CrossingListPage/DashboardCrossingListItem/StatusToggle/StatusToggle';
```

Are there other places where should implement this pattern for consistency?

Reply:
Yeah, there's a dozen places to update. We should decide on a pattern and submit a big PR to fix all of these at once.